### ISSUE TITLE
Button input sticks in Ableton

### ISSUE THREAD

**Issue by JPumphrey** [tommitytom/RetroPlug#9]
Using Ableton on Windows, there's an issue where the A button gets stuck down - i.e. the emulator receives the button press, but not the button release.

**Comment by JPumphrey** [comment]
As a workaround, you can set the hidden option `_ EnsureKeyMessagesForPlugins` in Ableton using these instructions https://help.ableton.com/hc/en-us/articles/209772865-Options-txt-file

**Comment by zeblade** [comment]
I am having the same problem. The key presses register but the releases don't. Only the arrow keys can be used normally. I have tried to use the "EnsureKeyMessagesForPlugins" option but it didn't work. I tried switching games, changing key asignment with no results. I did find a recent commit that @tommitytom posted (https://github.com/tommitytom/RetroPlug/commit/a13018f34027244e04333a5bb626b85848bf6e46) but I don't know what to do with the code.
